addons/account/models/company.py

```diff
@@ -122,8 +122,8 @@ class ResCompany(models.Model):
     transfer_account_code_prefix = fields.Char(string='Prefix of the transfer accounts')
     account_sale_tax_id = fields.Many2one('account.tax', string="Default Sale Tax", check_company=True)
     account_purchase_tax_id = fields.Many2one('account.tax', string="Default Purchase Tax", check_company=True)
-    account_sale_receipt_tax_id = fields.Many2one('account.tax', string="Default Sale Receipt Tax", check_company=True)
-    account_purchase_receipt_tax_id = fields.Many2one('account.tax', string="Default Purchase Receipt Tax", check_company=True)
+    account_sale_receipt_fiscal_position_id = fields.Many2one('account.fiscal.position', string="Default Sale Receipt Fiscal Position", check_company=True)
```

Comment:
Do we expect people to use this?
Especially considering that it can't be set in the UI